extensions/stackdriver/log/logger.cc

```diff
@@ -243,8 +260,16 @@ bool Logger::exportLogEntry(bool is_on_done) {
 
 void Logger::addTCPLabelsToLogEntry(
     const ::Wasm::Common::RequestInfo& request_info,
+    const ::Wasm::Common::FlatNode& peer_node_info,
     google::logging::v2::LogEntry* log_entry) {
   auto label_map = log_entry->mutable_labels();
+  setSourceCanonicalService(peer_node_info, label_map);
+  log_entry->set_text_payload(absl::StrCat(
+      label_map->find("source_canonical_service") != label_map->end()
+          ? (*label_map)["source_canonical_service"]
+          : flatbuffers::GetString(peer_node_info.workload_name()),
+      "-->",
```

Comment:
curious: was this specific text format requested by any of the users?  should we include spaces between the names and the `-->` ?

Reply:
no, we are adding this because otherwise tcp connection logs looks empty in the first look in stackdriver log viewer.

sure, will include spaces..

Reply:
Also, added a condition for destination in payload that if destination_canonical_service is not found, change to destination_service_name.